media/CircleEditor/view-sidebar.js

```diff
@@ -538,37 +575,232 @@ class NodeAttributeView {
                 this._element.appendChild(valueLine);
             }
         } else {
-            this._expander.innerText = '+';
-            while (this._element.childElementCount > 2) {
+            this._expander.className =
+                'sidebar-view-item-value-expander codicon codicon-chevron-down';
+            while (this._element.childElementCount > 3) {
                 this._element.removeChild(this._element.lastChild);
             }
         }
     }
 
-    /**
-     * TODO Implement edit()
-     * Attributes edit function
-     */
+    edit() {
+        while (this._element.childElementCount) {
+            this._element.removeChild(this._element.lastChild);
+        }
+        const type = this._attribute.type;
```

Comment:
`this._attribute.type` is used even `const type` is defined.
Either of them is fine, but just unify them :)